includes/types/CMB2_Type_Colorpicker.php

```diff
@@ -69,6 +57,44 @@ public function render( $args = array() ) {
 		return parent::render( $args );
 	}
 
+	/**
+	 * Sanitizes the given color, or array of colors.
+	 *
+	 * @since 2.7.1
+	 *
+	 * @param string|array $color The color or array of colors to sanitize.
+	 *
+	 * @return string|array The color or array of colors, sanitized.
+	 **/
+	public static function sanitize_color( $color ) {
+
+		if ( is_array( $color ) ) {
+
+			$color = array_map( 'CMB2_Type_Colorpicker::sanitize_color', $color );
```

Comment:
Keep callback consistent w/ other callbacks in CMB2.
```suggestion
			$color = array_map( array( 'CMB2_Type_Colorpicker', 'sanitize_color' ), $color );
```